chap-rationale.tex

```diff
@@ -261,6 +261,18 @@ \section{Capability Register File}
 might be quite different; effectively pinning the two to one another would
 reduce the efficiency of both.
 
+With a split register file, it may be useful for the architecture to add
```

Comment:
This change is fine, but I feel this section is now a bit out-dated overall as it still claims the use of split register files in general.  I hope I didn't add a separate rationale section on split vs merged in my earlier changes.  If so, that section and this need to be merged into one probably.

Reply:
Yes, that's true. This section definitely hints at us still using split register files, but a tiny bit of wordsmithing could probably change that, as it mostly all still applies.